client/packages/host/src/components/Footer/UserDetails.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FC } from 'react';
+import {
+  Box,
+  CircularProgress,
+  FlatButton,
+  PaperPopoverSection,
+  useAuthContext,
+  usePaperClickPopover,
+  useTranslation,
+  useNavigate,
+  useUserDetails,
+  useConfirmationModal,
+  RouteBuilder,
+  PowerIcon,
+  TextWithLabelRow,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+import { PropsWithChildrenOnly } from '@common/types';
+import { Tooltip } from 'recharts';
+
+export const UserDetails: FC<PropsWithChildrenOnly> = ({ children }) => {
+  const { logout, user, token } = useAuthContext();
+  const navigate = useNavigate();
+  const { hide, PaperClickPopover } = usePaperClickPopover();
+  const { isLoading } = useUserDetails(token);
+  const t = useTranslation();
+
+  const handleLogout = () => {
+    navigate(RouteBuilder.create(AppRoute.Login).build());
+    logout();
+  };
+
+  const showConfirmation = useConfirmationModal({
+    onConfirm: handleLogout,
+    message: t('messages.logout-confirm'),
+    title: t('heading.logout-confirm'),
+  });
+
+  const logoutButton = (
+    <FlatButton
+      startIcon={<PowerIcon fontSize="small" color="primary" />}
+      label={t('logout')}
+      disabled={false}
+      onClick={async () => {
+        hide();
+        showConfirmation();
+      }}
+      sx={{
+        whiteSpace: 'nowrap',
+        overflowX: 'hidden',
+        overflowY: 'visible',
+        textOverflow: 'ellipsis',
+      }}
+    />
+  );
+
+  return user && Tooltip ? (
```

Comment:
In regards of Tooltip. I'm not quite sure what it is, I've looked into but it seems to be a component? If it is a component, `user` alone should be sufficient 😃

Reply:
Ah good catch! Just a remnant from me troubleshooting!